test/Libraries/Microsoft.Extensions.AI.Tests/ChatCompletion/FunctionInvokingChatClientTests.cs

```diff
@@ -744,7 +744,7 @@ void AssertInvocationContext(FunctionInvocationContext context, int iteration, b
     }
 
     [Fact]
-    public async Task CanResumeFunctionCallingAfterTermination()
+    public async Task HaltFunctionCallingAfterTermination()
```

Comment:
Oh, cool. It looks like this test was likely failing before being fixed; right?

Reply:
Right, missed this one and adjusted for early termination.